### PR TITLE
Update language to make emeritus process clearer

### PR DESCRIPTION
**Pull request by dims**
incorporate review comments from @smarterclayton in https://github.com/kubernetes/community/pull/6240/files#r783993513

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: contributors/guide/owners.md [hunk 1]
```diff
@@ -118,17 +120,17 @@ emeritus_approvers:
...
-to prune inactive members from OWNERS files. A core principle in maintaining a
-healthy community is encouraging active participation. OWNERS have a higher
-activity requirement, as they directly impact the ability of others to contribute.
-If an owner should become inactive, they will be removed but may be added back
-should they become active again.
+to prune inactive folks from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. folks listed in OWNERS files
+have a higher activity requirement, as they directly impact the ability of others
+to contribute. If anyone listed in OWNERS files should become inactive, they will
+be removed but may be added back should they become active again.
```
Comment (smarterclayton) [inline contributors/guide/owners.md:127]: Are they removed, or added to emeritus_approvers?  Your code PR https://github.com/kubernetes/kubernetes/pull/107293/files seemed to move them to emeritus, but you seem to imply here they are completely removed.  Which is it?

Comment (mrbobbytables) [reply contributors/guide/owners.md:127]: reviewers are completely removed, approvers are bumped to `emeritus_approvers`.

Comment (dims) [reply contributors/guide/owners.md:127]: I'll add this in a minute

Comment (dims) [reply contributors/guide/owners.md]: Updated PR

File: contributors/guide/owners.md [hunk 2]
```diff
@@ -118,17 +120,19 @@ emeritus_approvers:
...
-An inactive OWNER is defined as someone with less than 10 Devstats recorded
-contributions within the past year, as shown by this [dashboard]. This is a
-conservative metric but should ensure only the removal of the most inactive
-OWNERS.
+to prune inactive folks from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. folks listed in OWNERS files
+have a higher activity requirement, as they directly impact the ability of others
+to contribute. If anyone listed in OWNERS files should become inactive, here is
+what we will do:
+- if the person is in reviewers section, their GitHub id will be removed from the section
```
Comment (smarterclayton) [inline contributors/guide/owners.md:128]: Great, exactly what I was looking for, thanks!

**Comment by smarterclayton** [comment]
Changes resolved my concerns, thank you.  Will let bobby tag.

**Comment by dims** [comment]
@smarterclayton @mrbobbytables can one of you please `/lgtm`?

File: contributors/guide/owners.md [hunk 3]
```diff
@@ -118,17 +120,19 @@ emeritus_approvers:
...
-If an owner should become inactive, they will be removed but may be added back
-should they become active again.
-
-
-An inactive OWNER is defined as someone with less than 10 Devstats recorded
-contributions within the past year, as shown by this [dashboard]. This is a
-conservative metric but should ensure only the removal of the most inactive
-OWNERS.
+to prune inactive folks from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. folks listed in OWNERS files
```
Comment (mrbobbytables) [inline contributors/guide/owners.md:124]:
```suggestion
healthy community is encouraging active participation. Those listed in OWNERS files
```
nit: could also do f -> F

**Review by mrbobbytables — COMMENTED** [review]
One small nit I saw, but otherwise lgtm

**Comment by dims** [comment]
@mrbobbytables applied your suggestion. now this is ready

**Review by mrbobbytables — APPROVED** [review]
Thanks!

/lgtm

**Review by Mxdon123 — APPROVED** [review]

**Pull request merged**